### ISSUE TITLE
Study 'exit page' - "Report" button and study rating button

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1057]
# TL;DR

At the end of a study, participants should see a screen that explains that the individual study is over, and they are interacting with Lookit/CHS again. Participants should be given the option here to 'report this study' if something has gone badly wrong. 

Note: Need to sort out how this interacts with exit URLs, particularly for labs that use it to redirect to the second half of a survey!

# Narrative

As the platform moves away from exhaustive centralized reviewing, we want to give participants easy ways to communicate when things are going well or poorly. 

<details>
<summary>Feature Request fields</summary>
<p>
**With this `details` wrapper, you can just use this ticket to record your findings.
When you're ready to turn this into an implementation request, delete the scoping label and
this wrapper, leaving only the filled out information.**

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests for new feature
- [ ] QA on new feature
- [ ] Sign off from Kim

# Implementation Notes
Leave important notes/suggestions here for implementation (important files, suggested approaches, etc.)

</p>
</details>
